meta-balena-common/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -92,6 +84,12 @@ do_install_append() {
 
     install -d ${D}${systemd_unitdir}/system
     install -m 0644 ${WORKDIR}/dev-zram0.swap ${D}${systemd_unitdir}/system/dev-zram0.swap
+
+    # Do not start getty on production mode
+    sed -i '/^\[Unit\]/a ConditionPathExists=/var/volatile/development-features' ${D}${systemd_unitdir}/system/getty@.service
+    sed -i '/^\[Unit\]/a PartOf=development-features.target' ${D}${systemd_unitdir}/system/getty.target
+    sed -i '/^\[Unit\]/a After=development-features.service' ${D}${systemd_unitdir}/system/getty.target
+    sed -i '/^\[Unit\]/a Requires=development-features.service' ${D}${systemd_unitdir}/system/getty.target
```

Comment:
Same as above, this may be better as a drop-in unit file rather than injection via sed, but no need to hold up this PR for it.

Reply:
As I was already using`sed` for the unit templates I decided to make all changes in the same place.

Reply:
```
/lib/systemd/system/getty.target.d/development-features.conf
```
```
[Unit]
ConditionPathExists=/var/volatile/development-features
PartOf=development-features.target
After=development-features.service
Requires=development-features.service
```
Up to you if you want to make the change for this PR as it would require retesting, but we could make the change in a followup PR.